lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -517,17 +525,15 @@ def _static_scrape_config(self, relation):
 
         hosts = self._relation_hosts(relation)
 
-        labeled_job_configs = []
-        for job in scrape_jobs:
-            config = self._labeled_static_job_config(
+        return [
+            self._labeled_static_job_config(
                 _sanitize_scrape_configuration(job),
                 job_name_prefix,
                 hosts,
                 scrape_metadata,
             )
-            labeled_job_configs.append(config)
-
-        return labeled_job_configs
```

Comment:
Similar comments elsewhere in this review. I do not think there is any reason to replace a  `return labeled_job_configs` which makes clear what is being returned by naming it. The suggested replacement of a nested list comprehension with nested function calls that nest function calls is not a readability improvement or a required performance gain, if any.